lighthouse-assert/test/fixtures/expectations/default-expectations.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/**
+ * Default expected Lighthouse audit values for tests
+ */
+export default [
+  {
+    initialUrl: 'http://localhost:10200/tricky-ttci.html',
+    url: 'http://localhost:10200/tricky-ttci.html',
+    audits: {
+      'first-contentful-paint': {
+        score: {
+          "warn": '>=65',
+          "error": '<=85',
+        },
+        rawValue: '>3000',
+      },
+      'first-meaningful-paint': {
+        score: {
+          "warn": '<=65',
```

Comment:
I'm a little confused by the values that have `warn` less than the `error` value, shouldn't it be the other way around?

Reply:
Yeah, of course. My bad. Thx 👍